main.go

```diff
@@ -85,16 +90,30 @@ const (
 )
 
 func getConfig() (*config, error) {
+	var reposSyncList []string
 	dryRunMode := os.Getenv("DRY_RUN") != ""
 	githubSecret := os.Getenv("GITHUB_SECRET")
 	githubToken := os.Getenv("GITHUB_TOKEN")
 	gitlabToken := os.Getenv("GITLAB_TOKEN")
 	gitlabBaseURL := os.Getenv("GITLAB_BASE_URL")
 	integrationDirectory := os.Getenv("INTEGRATION_DIRECTORY")
-	logLevel, found := os.LookupEnv("INTEGRATION_TEST_RUNNER_LOG_LEVEL")
 
-	logrus.SetLevel(logrus.InfoLevel)
+	//
+	// Currently we don't have a distinguishment between GitHub events and features.
+	// Different features might be implemented across different events, but in future
+	// it's probability that we might implement proper features selection. For now the
+	// straight goal is to being able to configure the runner to only sync repos on
+	// push events and disable all the rest (to be used by the CFEngine team).
+	//
+	// default: process push events (sync repos) if not explicitly disabled
+	isProcessPushEvents := os.Getenv("DISABLE_PUSH_EVENTS_PROCESSING") == ""
```

Comment:
I guess it is the same, but we could use something like
```suggestion
	_ , isProcessPushEvents := os.LookupEnv("DISABLE_PUSH_EVENTS_PROCESSING")
```

In both cases it has the gotcha that setting the variable to `false` or some other negative meaning string still validates to "true". Anyway as we will redesign this part is okey to keep it simple for now :+1:

Reply:
This is a little bit tricky, because it sets reverse value. So, if the variable is empty then we process the event. This is needed to keep default behaviour with no extra settings for the current deployments. With `os.LookupEnv` the value is not reversed and additional settings will require for the current deployments to keep the events enabled.

Reply:
Thank you for the explanation.